direct/src/dist/pip_wrapper.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+
+import platform
+import pip
+import sys
+import os
+
+target_platform = None
+sys_platform = None
+platform_machine = None
+platform_system = None
+os_name = None
+
+for (index, arg) in enumerate(sys.argv):
+    if arg == '--platform':
+        target_platform = sys.argv[index + 1]
+
+if target_platform is not None:
+    if target_platform.startswith("win"):
+        sys_platform = 'win32'
+        os_name = 'nt'
+    elif target_platform.startswith("manylinux") or target_platform.startswith("linux"):
```

Comment:
It would probably be safe enough to just do `'linux' in target_platform`

Reply:
Indeed,  will be much clearer :-)